### ISSUE TITLE
遇到--fork timeout的问题

### ISSUE THREAD

**Issue by zhanghuidinah** [aylei/kubectl-debug#90]
- fork 时 timeout

![image](https://user-images.githubusercontent.com/31201589/71245731-50500a00-2350-11ea-9607-8e9106dc6537.png)

- `意外` 不可避免，也可以退出时回收？不然还要来 debug 这个 pod 的问题
<img width="831" alt="截屏2019-12-20下午5 45 05" src="https://user-images.githubusercontent.com/31201589/71245917-b2107400-2350-11ea-896d-0d3aed28e6c0.png">

**Comment by aylei** [comment]
Good catch! The forked Pod should also be automatically deleted on wait time out.
